packages/core/dialect/mssql/queries.ts

```diff
@@ -101,8 +101,7 @@ SELECT
   concat(
     f.routine_schema,
     '.',
-    f.routine_name,
-    concat('(', STRING_AGG(p.data_type, ','), ')')
+    f.routine_name    
```

Comment:
I changed `signature` to be without arguments, because I needed it for tooltip, etc. in case func/proc is inside a package, e.g.: ${schema}.${package}.${procedure}.

Reply:
not tooltip but mainly for snippet :)

Reply:
@mickeypearce was this change intentional? I see it in mssql file. I understand the change and it does make sense, but if it's intentional, I'll need to add to other dialects as well.

Reply:
Yes, I missed the other dialects, this should be it.